### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Wiecek-K**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,6 +22,52 @@
...
+You can access the live version of the API at: [http://13.50.5.104:3000](http://13.50.5.104:3000)
+
+## Base Endpoints
+
+### 1. Create User
+
+- **Endpoint:** `/users`
+- **Method:** `POST`
+- **Request Body:**
+
```
Comment (NerdbordMentor) [inline README.md:38]: **issue**: Duplicate section for 'Login' endpoint. Consider removing one to avoid confusion.

**Review by NerdbordMentor — CHANGES_REQUESTED** [review]
## Overall Feedback

The code changes include the addition of API documentation to the `README.md` file, which details the live version of the API and base endpoints for user creation and login. The URL to the deployed service is included, which meets one of the task requirements. However, there is no evidence in the code changes summary or the actual code changes that a Docker container was created and deployed, which is a critical requirement.

### Things Missing:

- [ ] Creation and deployment of a Docker container.

### Things That Went Well:

- The API documentation in the `README.md` is clear and provides necessary details for accessing the live API and using the base endpoints.

File: README.md [hunk 2]
```diff
@@ -22,6 +22,51 @@
...
```
Comment (NerdbordMentor) [inline README.md:38]: **praise**: Well-structured documentation for the 'Generate Short URL' endpoint.

**Review by NerdbordMentor — CHANGES_REQUESTED** [review]
## Overall Feedback

The code changes include the addition of API documentation to the README file, which details the live version of the API and the base endpoints for creating a user, logging in, and generating a short URL. The task requirements specify that a docker container must be deployed and the URL to the deployed service must be included in the `readme.md` file. The provided URL in the README file suggests that the service is deployed, which meets the task requirement. However, there is no explicit mention of the docker container deployment in the code changes summary or the actual code changes.

### Things Missing:

- [ ] Explicit confirmation or documentation of the docker container deployment.

### Things That Went Well:

- The API documentation is clear and well-structured, providing necessary details for users to interact with the API.
- The live version URL is included, which is a critical requirement for the task.

**Review by Sumick — APPROVED** [review]
Very good job 🫡

**Pull request merged**